ecs-cli/modules/cli/compose/project/project_parseV3.go

```diff
@@ -11,20 +17,93 @@ import (
 func (p *ecsProject) parseV3() (*[]containerconfig.ContainerConfig, error) {
 	logrus.Debug("Parsing v3 project...")
 
-	// TODO: parse v3, convert ServiceConfigs to ContainerConfigs
-	configDetails := types.ConfigDetails{}
+	// load and parse each compose file into a configFile
+	configFiles := []types.ConfigFile{}
+	for _, file := range p.ecsContext.ComposeFiles {
+
+		loadedFile, err := ioutil.ReadFile(file)
+		if err != nil {
+			return nil, err
+		}
+		parsedFile, err := loader.ParseYAML(loadedFile)
+		if err != nil {
+			return nil, err
+		}
+		configFile := types.ConfigFile{
+			Filename: file,
+			Config:   parsedFile,
+		}
+		configFiles = append(configFiles, configFile)
+	}
+
+	// get build environment
+	env, err := getLocalEnv()
+	if err != nil {
+		return nil, err
+	}
+
+	// get compose working dir
+	wrkDir, err := getWorkingDir(p.ecsContext.ComposeFiles[0])
+	if err != nil {
+		return nil, err
+	}
+
+	// build config details
+	configDetails := types.ConfigDetails{
+		WorkingDir:  wrkDir,
+		ConfigFiles: configFiles,
+		Environment: env,
+	}
+
+	// load config from config details
```

Comment:
Nit: In generally I like having comments to explain steps of a process, but in this case I feel like the code documents itself pretty well, so could do without the comment. Unless there are more relevant deatils about loading the config that you wanted to make explicit? (same with other code comments in here)

Reply:
I'm ok with removing some of the comments here - they're mostly a holdover from when i was scaffolding out what needed to be done